drone-localization/main.py

```
@@ -1,13 +1,47 @@
+    version="0.1.0"
+)
 
-# Подключаем роутеры
-app.include_router(localize.router, prefix="/api")
+# ✅ CORS для Streamlit (порт 8501)
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=[
+        "http://localhost:8501",
+        "http://127.0.0.1:8501",
+        "*"  # Для разработки. В продакшене убрать!
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `В` (CYRILLIC CAPITAL LETTER VE). Did you mean `B` (LATIN CAPITAL LETTER B)?

(RUF003)